.github/workflows/conformance-gke-v1.11.yaml

```diff
@@ -193,7 +193,7 @@ jobs:
             --machine-type e2-custom-2-4096 \
             --disk-type pd-standard \
             --disk-size 10GB \
-            --node-taints node.cilium.io/agent-not-ready=true:NoExecute \
+            --node-taints ignore-taint.cluster-autoscaler.kubernetes.io/cilium-agent-not-ready=true:NoExecute \
```

Comment:
Same as above, let's reset all changes to this file until the backport PR is open.